Extensions/image/package.lisp

```diff
@@ -0,0 +1,63 @@
+
+(defpackage :mcclim-image
+  (:use #:clim #:clim-lisp)
+
+  (:import-from #:clim-internals
+                #:design
+                #:pattern
+                #:with-medium-options
+                #:pattern-width
+                #:pattern-height
+                #:opticl-read-bitmap-file
+                #:medium-draw-pattern*)
+  (:export
+   #:image
+   #:rgb-image
+   #:rgb-pattern
+   #:image-width
+   #:image-height
+   #:image-data
+   #:image-alpha-p
+   #:mage-alpha-p
+   #:rgb-image-design
+   #:make-rgb-image-design
+   #:medium-free-image-design
+   #:medium-draw-image-design*
+   #:sheet-rgb-image
+   #:sheet-rgb-data
+   #:draw-design
+   #:xpm-parse-file
+   #:make-pattern-from-bitmap-file
+   #:medium-data
+   ))
+
+(in-package :mcclim-image)
+
+;;;
+;;; import symbols into climi
+;;;
+
```

Comment:
> can't these symbols be imported in :import-from clause in the defpackage above?
No, I want an ':import-to'.

These symbols are imported into clim-internals to make the old code running. I hope to be able to fix the situation in the future.

I think the correct place for the public API is in the 'clim-extensions' package.